README.md

```diff
@@ -47,11 +47,8 @@
 
```

Comment:
- [ ] It would be nice if you ask your partner @ethred to make some commits and push some code in this pull request to follow the Pair-programming rule properly 👍 I can his commit but he didn't push any code 🤔 It would be great if you push some code such as tests to follow the project requirements 😊

Reply:
Sure, Thanks a lot